### ISSUE TITLE
HPOOL DİSCORD ACCOUNT

### ISSUE THREAD

**Issue by alikorayozcilingir** [hpool-dev/chia-miner#320]
I CAN'T FİND A DİSCORD ACCOUNT AT HPOOL ?

**Comment by tokenhoki** [comment]
curious.....
no twiter no discord they dont have anything, i already contack by email never respond.. wtf with this pool

**Comment by HailongZhu** [comment]
This is a Chinese pool, so use wechat.

**Comment by alikorayozcilingir** [comment]
You are right, but the wechat application does not allow us to connect. We cannot find the reference person. That's why we created our own discord channel. The authorities can make any disclosure to us globally from there.

https://discord.gg/daz9VUTH
